README.md

```diff
@@ -226,6 +226,14 @@ index them run:
 make reindex
 ```
 
+### 2.6 Clearing SQS queues
+
+1. Purge Notification Queue
+2. Disable event binding document queue and azul-indexer-write lambda
```

Comment:
```suggestion
2. Disable the event binding between the document queue and `azul-indexer-…-write` lambda
```

Explain quickly how to do that. Name the document queue. Use `…` to skip over the deployment part in the queue name.